app/workers/slack_notifier_worker.rb

```diff
@@ -0,0 +1,16 @@
+class SlackNotifierWorker
+  include Sidekiq::Worker
+
+  def perform
+    User.all.active.each do |user|
+      if user.slack_id != "not imported" && user.total_debt.to_i < 50 && user.total_debt.to_i != 0
```

Comment:
I'd get rid of `not imported` default and probably go with `if user.slack_id && ...`. Also: WDYTA moving  the hardcoded `50` into some constant?